analyticsdataserver/settings/base.py

```diff
@@ -49,6 +49,11 @@
     }
 }
 ########## END DATABASE CONFIGURATION
+ELASTIC_SEARCH_LEARNERS_HOST = None
```

Comment:
Nit: I'd remove the first underscore in each name to be: `"ELASTICSEARCH_LEARNERS_HOST"`.